kogito-editors-js/packages/boxed-expression-component/src/__tests__/components/LogicTypeSelector/LogicTypeSelector.test.tsx

```diff
@@ -86,32 +86,6 @@ describe("Logic type selection", () => {
     expect(baseElement.querySelector(".logic-type-selector")).toBeTruthy();
     expect(baseElement.querySelector(".logic-type-selector")).toHaveClass("logic-type-selected");
   });
-
-  test("should reset the selection, when logic type is selected and clear button gets clicked", async () => {
-    const expression = { name: "Test", logicType: LogicType.LiteralExpression, dataType: DataType.Undefined };
-
-    const { baseElement } = render(
-      usingTestingBoxedExpressionI18nContext(
-        <LogicTypeSelector
-          selectedExpression={expression}
-          getPlacementRef={() => document.body as HTMLDivElement}
-          onLogicTypeResetting={_.identity}
-          onLogicTypeUpdating={_.identity}
-        />
-      ).wrapper
-    );
-
-    await triggerContextMenu(baseElement as HTMLElement, ".logic-type-selector");
-
-    act(() => {
-      const clearButtonElement = baseElement.querySelector(".context-menu-container button")!;
-      const clearButton = clearButtonElement as HTMLButtonElement;
-      clearButton.click();
-    });
-
-    expect(baseElement.querySelector(".logic-type-selector")).toBeTruthy();
-    expect(baseElement.querySelector(".logic-type-selector")).toHaveClass("logic-type-not-present");
-  });
```

Comment:
why this test case is not needed anymore?

Reply:
This test was moved to the "BoxedExpressionEditor". The LogicTypeSelector doesn't have an internal state saving the current logicType anymore, as this is managed by its parent component. But after your review, I've made some modifications and moved it back to the LogicTypeSelector tests.